schema/deploy/shipping/views.sql

```diff
@@ -930,4 +930,59 @@ comment on view shipping.scan_return_results_v1 is
   'View of barcodes and presence/absence results for SCAN return of results on the UW Lab Med site';
 
 
+create or replace view shipping.reportable_hcov19_v1 as
+
+    select
+        presence_absence.presence_absence_id,
+        organism.lineage::text,
+        sample.identifier as sample,
+        sample_id.barcode as sample_barcode,
+        collection_id.barcode as collection_barcode,
+        site.identifier as site,
+        presence_absence.details->>'reporting_log' as reporting_log,
+        sample.details->'_provenance'->>'workbook' as workbook,
+        sample.details->'_provenance'->>'sheet' as sheet,
+        sample.details->>'sample_origin' as sample_origin,
+        sample.details->>'swab_site' as swab_site,
+        encounter.details ->> 'language' as language
+
+    from warehouse.presence_absence
+    join warehouse.target using (target_id)
+    join warehouse.organism using (organism_id)
+    join warehouse.sample using (sample_id)
+    join warehouse.identifier sample_id on (sample.identifier = cast(sample_id.uuid as text))
+    join warehouse.identifier collection_id on (sample.collection_identifier = cast(collection_id.uuid as text))
```

Comment:
This join requires the sample to have a collection identifier, and although I expect that to always be true, we probably shouldn't assume lest we miss reportables due to a manifest data problem. (Because p/a results can come in for a sample without a collection identifier.)

Reply:
Can we even report these results back to participants if there are not two identifiers to verify?